lua/ttt2/libraries/database.lua

```diff
@@ -1139,14 +974,14 @@ if SERVER then
 	---
 	-- This is called upon receiving a get request from a player to send a value back
 	-- @warning Dont use this function if you want to get a value from the database, this is meant to be used internally for a client request
-	-- @param table requestData = {plyID64, identifier, index, itemName, key} contains player and the data they requested
+	-- @param table requestData = {identifier, index, itemName, key} contains player and the data they requested
```

Comment:
as an example, this looks better to me because there is actual explanation there

Reply:
the general idea was to give a c-style class explanation whats inside that variable. THey are mostly internal and shall help to understand what you can use.

Reply:
probably @Histalek should explain how it works best with his tool

Reply:
this is fine, if/when we switch to the luals annotations we will need to figure out alot of stuff anyways.

right now this is fine as every key in the table is documented and it will just be part of the parameter description. Best case would be a short description of each key, but i think that is to much to put into the parameter description as of now.

Reply:
additionally most of these (all?) are in `@internal` functions. so only people working on the database library would be exposed to these docs. And we have worse docs for internal stuff tbh